File: Samples/JotFormTest/Classes/Controller/GetFormPropertiesViewController.m
//
//  GetFormPropertiesViewController.m
//  JotFormTest
//
//  Created by Curtis Stilwell on 8/21/17.
//  Copyright © 2017 wang. All rights reserved.
//

#import "GetFormPropertiesViewController.h"
#import "SharedData.h"
#import "Common.h"

@interface GetFormPropertiesViewController ()

@property (nonatomic, weak) IBOutlet UITextView *textView;

@end

@implementation GetFormPropertiesViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view from its nib.
    [[SharedData sharedData].apiClient getFormProperties:FORM_ID onSuccess:^(id result) {
      
        self.textView.text = [NSString stringWithFormat:@"%@",result];
   
    } onFailure:^(NSError *error) {
        
    }];
}

@end

File: Samples/JotFormTest/Classes/Controller/GetFormPropertiesViewController.swift
//
//  GetFormPropertiesViewController.swift
//  JotFormTest
//
//  Created by Curtis Stilwell on 2/4/18.
//  Copyright © 2018 wang. All rights reserved.
//

import Foundation

class GetFormPropertiesViewController: UIViewController {
    @IBOutlet weak var textView: UITextView!
    
    override func viewDidLoad() {
        // Do any additional setup after loading the view from its nib.
        SharedData.sharedData.apiClient?.getFormProperties(FORM_ID, onSuccess: {(_ result: Any) -> Void in
            self.textView.text = "\(result)"
        }, onFailure: {(_ error: Error?) -> Void in
        })
    }
}
